src/core/text/Text.js

```diff
@@ -744,7 +744,7 @@ export default class Text extends Sprite
 
     set text(text) // eslint-disable-line require-jsdoc
     {
-        text = String(text || ' ');
+        text = String(text === '' || text === null ? ' ' : text);
```

Comment:
Here is the line that @andrewstart is suggesting. Requires disabling two rules in order to use the loose `null` comparison.
```js
text = String(text === '' || text == null ? ' ' : text); // eslint-disable-line no-eq-null, eqeqeq
```
I would suggest we use the more explicit (and shorter) version:
```js
test = String(text === '' || text === null || text === undefined ? ' ' : text);
```